candidate-testing.js

```diff
@@ -3,44 +3,59 @@ const input = require('readline-sync');
 // TODO 2: modify your quiz app to ask 5 questions //
 
 // TODO 1.1a: Define candidateName // 
-let candidateName;
+let candidateName = "";
 // TODO 1.2a: Define question, correctAnswer, and candidateAnswer //
-let question;
-let correctAnswer;
-let candidateAnswer;
-let questions;
-let correctAnswers;
-let candidateAnswers;
+let question = "Who was the first American woman in space? ";
+let correctAnswer = "Sally Ride";
+let candidateAnswer = (input.question(question + ""));
```

Comment:
Just so you know...When you run your program currently this is actually running before you ask for the candidates name.

Reply:
Yea that was due to my code entered on the canidateAnswer. Deleting that fixed that issue